docs/recipes.rst

```diff
@@ -577,100 +479,13 @@ Data formats which can be added are :-
-                        "size": "7",
-                        "type": "char"
-                    }
-                ]
-            },
-            "url": "https://ndownloader.figshare.com/files/11188550"
-        },
-        {
-            "dialect": {
-                "delimiter": ","
-            },
-            "name": "plots",
-            "path": "plots.csv",
-            "sqlite_data": ["plots","portal_project.sqlite"],
-            "schema": {
-                "fields": [
-                    {
-                        "name": "plot_id",
-                        "type": "int"
-                    },
-                    {
-                        "name": "plot_type",
-                        "size": "25",
-                        "type": "char"
-                    }
-                ]
-            },
-            "url": "https://ndownloader.figshare.com/files/11188550"
-        },
-        {
-            "dialect": {
-                "contains_pk": "True",
-                "delimiter": ",",
-                "missingValues": [
-                    "",
-                    null
-                ]
-            },
-            "name": "surveys",
-            "path": "surveys.csv",
-            "sqlite_data": ["surveys","portal_project.sqlite"],
-            "schema": {
-                "fields": [
-                    {
-                        "name": "record_id",
-                        "type": "int"
-                    },
-                    {
-                        "name": "month",
-                        "type": "int"
-                    },
-                    {
-                        "name": "day",
-                        "type": "int"
-                    },
-                    {
-                        "name": "year",
-                        "type": "int"
-                    },
-                    {
-                        "name": "plot_id",
-                        "type": "int"
-                    },
-                    {
-                        "name": "species_id",
-                        "size": "2",
-                        "type": "char"
-                    },
-                    {
-                        "name": "sex",
-                        "size": "1",
-                        "type": "char"
-                    },
-                    {
-                        "name": "hindfoot_length",
-                        "type": "double"
-                    },
-                    {
-                        "name": "weight",
-                        "type": "double"
-                    }
-                ]
-            },
     ...
 
 
-4. GeoJSON Data :- To make a script for GeoJSON data add object ``geojson_data`` objects as shown in the script ``lake-county-illinois-cancer-rates`` for `Cancer Rates Lake County` dataset. Add the name of the GeoJSON file in the ``geojson_data`` object.
+4. GeoJSON Data :- To make a script for GeoJSON data add object ``geojson_data``
```

Comment:
GeoJSON Data :-  Add `geojson_data` key to the resource section. The value ...

Reply:
Do this for the rest of the descriptions above.

We basically want to make the docs short and to the point. In this section, we are talking about the ways to add the kind of raw data format, So we can go ahead and remove the part `To make a script for`